File: 01_Semestre/IDS/05/Ex07.py
sBase = float(input("Digite su sueldo base: "))
venta1 = float(input("Digite el valor de su primera venta: "))
venta2 = float(input("Digite el valor de su segunda venta: "))

comisiones = (venta1 + venta2)*0.15
sFinal = sBase + comisiones

print("Su ganancia en comisiones es de", comisiones)
print("Su ganancia total es de", sFinal)

File: 01_Semestre/IDS/05/Ex07.java
import java.util.Scanner;

public class Ex07 {
   public static void main (String[] args) {
      Scanner sc = new Scanner(System.in);
      double sBase, venta1, venta2, comisiones, sFinal;

      System.out.print("Digite su sueldo base: ");
      sBase = sc.nextDouble();
      System.out.print("Digite el valor de la primera venta: ");
      venta1 = sc.nextDouble();
      System.out.print("Digite el valor de la segunda venta: ");
      venta2 = sc.nextDouble();

      comisiones = (venta1 + venta2)*0.15;
      sFinal = sBase + comisiones;

      System.out.println("Su ganancia en comisiones es de " + comisiones);
      System.out.println("Su sueldo final es de " + sFinal);
   }
}
